path.go

```diff
@@ -119,12 +119,8 @@ var (
 	routeDelimiter = []byte{slashDelimiter, '-', '.'}
 	// list of greedy parameters
 	greedyParameters = []byte{wildcardParam, plusParam}
-	// list of chars for the parameter recognizing
```

Comment:
@ksw2000 You can keep the new Boolean lists outside the functions in the layer so that the configuration is in one place and expansion is easy.

Hopefully, this should not make any difference to performance.